content/tt/paka.md

```diff
@@ -1,22 +1,20 @@
 +++
 title = "PAKA"
 template = "team_page.html"
-authors = ["Szymon Iwulski"]
+authors = ["Szymon Iwulski", "Sewi The Referee"]
 [extra]
 toclevel = 3
 orgs = ["ptw"]
 [extra.gallery]
 1 = { path = "paka.webp", caption = "[Taras](@/w/taras.md), [Boro](@/w/boro.md) and [Disco Pablo](@/w/disco-pablo.md), with their [PTW Tag Team Championships](@/c/ptw-tag-team-championship.md).", source = "Official PTW Facebook" }
 +++
 
-PAKA (roughly _The PACK_) were a babyface tag team, active in [Prime Time Wrestling](@/o/ptw.md). As a three-man team, the members saw success both in tag-team and singles competition.
-
-## Basic info
-
-* Years active: 2021-2024
+PAKA (roughly _The PACK_) were a babyface tag team, active in [Prime Time Wrestling](@/o/ptw.md), and later recognized both in [Legacy of Wrestling](@/o/low.md) and [Maniac Zone Wrestling](@/o/mzw.md). As a three-man team, the members saw success both in tag-team and singles competition.
 
 ## History
 
+#### Prime Time Wrestling (2021-2024)
```

Comment:
Incorrect header level: following `## History` we should have `### PTW` and then `#### Debut etc`, and this needs to be consistently applied to headers below. Otherwise, this will cause a very confusing table of contents.